applications/GeoMechanicsApplication/custom_constitutive/coulomb_yield_surface.h

```diff
@@ -28,7 +28,7 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) CoulombYieldSurface : public YieldSu
 
     CoulombYieldSurface(double FrictionAngleInRad, double Cohesion, double DilatationAngleInRad);
 
-    [[nodiscard]] double YieldFunctionValue(const Vector& rPrincipalStress) const override;
+    [[nodiscard]] double YieldFunctionValue(const Vector& rSigmaTau) const override;
     [[nodiscard]] Vector DerivativeOfFlowFunction(const Vector& rPrincipalStress) const override;
```

Comment:
As per the previous comment, let's "unname" this function parameter:
```suggestion
    [[nodiscard]] Vector DerivativeOfFlowFunction(const Vector&) const override;
```

Reply:
Done.